modules/video/include/opencv2/video/tracking.hpp

```diff
@@ -250,7 +250,8 @@ when fullAffine=false.
 @sa
 estimateAffine2D, estimateAffinePartial2D, getAffineTransform, getPerspectiveTransform, findHomography
  */
-CV_EXPORTS_W Mat estimateRigidTransform( InputArray src, InputArray dst, bool fullAffine );
+CV_EXPORTS_W Mat estimateRigidTransform( InputArray src, InputArray dst, bool fullAffine, int RANSAC_MAX_ITERS=500, double RANSAC_GOOD_RATIO=0.5,
+                                         int RANSAC_SIZE0=3);
```

Comment:
In OpenCV 3.x we preserve binary compatibility. So you need to add another function with those extra parameters.